### ISSUE TITLE
Feature inquiry: using a combination of both WM_NAME and WM_CLASS

### ISSUE THREAD

**Issue by 18kimn** [roosta/i3wsr#31]
I'm guessing this is out of scope, but I'm wondering if there's a way to specify the `wm_property` option on a per-application basis -- so I could have WM_NAME be used for terminal apps, as it's helpful to know what's running in it, and WM_CLASS be used in other situations when the WM_NAME property is too verbose and cannot be simplified via regex rules

A possibly more implement-able feature could be specifying whether to use WM_CLASS or WM_NAME in the `[aliases]` section to accomplish the same task. E.g. there could be a main `[aliases]` section that would rename based on `wm_property` as it does now, and users could also have a section `[aliases.wm_class]` to rename based on WM_CLASS regardless of whatever X window property is specified in `wm_property`

please let me know what your thoughts are! Thank you!

**Comment by roosta** [comment]
I'll look at this and see how much work it is to implement, your idea on a possible impl is pretty solid. Unfortunately it goes in the TODO list for now, I've too much on my plate to implement new features at the moment.

**Comment by roosta** [comment]
So I set this up on the develop branch, there are no docs yet, but I'll summarize here. just remember to `git checkout develop`. 

I've removed the `wm_propery` option in favor of alias sub tables. So what was `[aliases]`, is now be split into `[aliases.class]`, `[aliases.instance]`, and `[aliases.name]`.

i3wsr will try to check name for aliases first, then instance, then class, and will lastly fall back to no alias just showing class.

To get the old behavior just pick whatever table matches your wm_property option.

Let me know if this works for you.
